archive/rpm.ksy

```diff
@@ -0,0 +1,472 @@
+    1064: rpmversion
+    1065: triggerscripts # from lib/rpmtag.h
+    1066: triggername # from lib/rpmtag.h
+    1067: triggerversion # from lib/rpmtag.h
+    1068: triggerflags # from lib/rpmtag.h
+    1069: triggerindex # from lib/rpmtag.h
+    1080: changelogtime
+    1081: changelogname
+    1082: changelogtext
+    1085: preinstall_interpreter # /bin/sh
+    1086: postinstall_interpreter # /bin/sh
+    1087: preuninstall_interpreter # /bin/sh
+    1088: postuninstall_interpreter # /bin/sh
+    1089: buildarchs # from lib/rpmtag.h
+    1090: obsoletename
+    1092: triggerscriptprog # from lib/rpmtag.h
+    1094: cookie
+    1095: filedevices
+    1096: fileinodes
+    1097: filelangs
+    1106: sourcepackage # from lib/rpmtag.h
+    1112: provideflags
+    1113: provideversion
+    1114: obsoleteflags
+    1115: obsoleteversion
+    1116: dirindexes
+    1117: basenames
+    1118: dirnames
+    1122: optflags
+    1123: disturl
+    1124: payload_format
+    1125: payload_compressor
+    1126: payload_flags
+    1127: installcolor # from lib/rpmtag.h
+    1128: installtid # from lib/rpmtag.h
+    1129: removetid # from lib/rpmtag.h
+    1131: rhnplatform
+    1132: platform
+    # below are all from lib/rpmtag.h
+    1140: filecolors
+    1141: fileclass
+    1142: classdict
+    1143: filedependsx
+    1144: filedependsn
+    1145: dependsdict
+    1146: sourcepkgid
+    1148: fscontexts
+    1149: recontexts
+    1150: policies
+    1151: pretrans
+    1152: posttrans
+    1153: pretransprog
+    1154: posttransprog
+    1155: disttag
+    1195: dbinstance
+    1196: nvra
+    5000: filenames
+    5001: fileprovide
+    5002: filerequire
+    5005: triggerconds
+    5006: triggertype
+    5007: origfilenames
+    5008: longfilesizes
+    5009: longsize
+    5010: filecaps
+    5011: filedigestalgo
+    5012: bugurl
+    5013: evr
+    5014: nvr
+    5015: nevr
+    5016: nevra
+    5017: headercolor
+    5018: verbose
+    5019: epochnum
+    5020: preinflags
+    5021: postinflags
+    5022: preunflags
+    5023: postunflags
+    5024: pretransflags
+    5025: posttransflags
+    5026: verifyscriptflags
+    5027: triggerscriptflags
+    5030: policynames
+    5031: policytypes
+    5032: policytypesindexes
+    5033: policyflags
+    5034: vcs
+    5035: ordername
+    5036: orderversion
+    5037: orderflags
+    5040: instfilenames
+    5041: requirenevrs
+    5042: providenevrs
+    5043: obsoletenevrs
+    5044: conflictnevrs
+    5045: filenlinks
+    5046: recommendname
+    5047: recommendversion
+    5048: recommendflags
+    5049: suggestname
+    5050: suggestversion
+    5051: suggestname
+    5052: supplementname
+    5053: supplementversion
+    5054: supplementflags
+    5055: enhancename
+    5056: enhanceversion
+    5057: enhanceflags
+    5058: recommendnevrs
+    5059: suggestnevrs
+    5060: supplementnevrs
+    5061: enhancenevrs
+    5062: encoding
+    5066: filetriggerscripts
```

Comment:
please format the ids properly. In order to use `-orig-id` you would have to convert it to extended enum value, i.e.:

```yaml
5066:
  id: file_trigger_scripts
  -orig-id: file_trigger_scripts
```

Reply:
added -orig-ids everywhere, should be clearer now.